public-ui/src/app/views/ehr-view/ehr-view.component.html

```diff
@@ -150,7 +157,11 @@ <h6 *ngIf="searchText.value" class="primary-display medline-link">Interested in
                   <ng-template #code_match_not_standard>
                     <div class="body-lead info-text tbl-d first">
                       <div class="code-tooltip">
-                        <app-highlight-search [indNum]="i+1" [text]="r.conceptName" [searchTerm]="prevSearchText">
+                          <span *ngIf="totalResults > searchRequest.maxResults" class="item-index">
```

Comment:
if the search is less than max results don't do anything special else reformat count based on current page